### PR TITLE
blank when no options

### PR DESCRIPTION
**Pull request by Jesus82**
I left it when there are no friends, let me know if you want me to add any translation key.

### CODE REVIEW

**Comment by lucaslcode** [comment]
Yes I think it does need a string. Maybe "Couldn't find any friends". 
Also make sure to run `yarn format` or set up your IDE to format on save (should be double quote marks)

File: app/frontend/src/features/messages/groupchats/CreateGroupChat.tsx [hunk 1]
```diff
@@ -148,7 +148,7 @@ export default function CreateGroupChat({ className }: { className?: string }) {
                     multiple={true}
                     loading={friends.isLoading}
                     options={friends.data ?? []}
-                    noOptionsText={''}
+                    noOptionsText={"Couldn't find any friends"}
```
Comment (lucaslcode) [inline app/frontend/src/features/messages/groupchats/CreateGroupChat.tsx]: Could you put it in the nearest `constants.ts`?

**Review by lucaslcode — APPROVED** [review]

**Comment by Jesus82** [comment]
Hi, I see that 1 check failed in the pipeline, but the PR was merged. Should I delete the branch?

**Comment by lucaslcode** [comment]
The failed part was just some coverage calculator, looks like a one-off server error.
Since the PR is merged now, I've deleted the branch (would need a revert if there was a problem).

**Pull request merged**